test/evergreen_develop.yml

```diff
@@ -18,7 +18,7 @@ buildvariants:
       LD_LIBRARY_PATH=$WT_BUILDDIR:$WT_BUILDDIR/test/utility/
     CC_OPTIMIZE_LEVEL: -DCC_OPTIMIZE_LEVEL=-O3
     HAVE_DIAGNOSTIC: -DHAVE_DIAGNOSTIC=0
-    CMAKE_TOOLCHAIN_FILE: -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/mongodbtoolchain_v4_gcc.cmake
+    CMAKE_TOOLCHAIN_FILE: -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/mongodbtoolchain_stable_gcc.cmake
     CMAKE_INSTALL_PREFIX: -DCMAKE_INSTALL_PREFIX=$(pwd)/cmake_build/LOCAL_INSTALL
     python_binary: '/opt/mongodbtoolchain/v4/bin/python3'
     pip3_binary: '/opt/mongodbtoolchain/v4/bin/pip3'
```

Comment:
With `CMAKE_TOOLCHAIN_FILE` lined up to a "stable" version, for consistency and ease of maintenance, other expansions set to "v4" toolchain (e.g. PATH, ASAN_SYMBOLIZER_PATH, besides the 2 here) better adopt a similar approach by using the stable version instead of a hard-coded version. I'm fine if a follow-on ticket covers that part of the work.

Reply:
That sounds sensible to me. I'll make a ticket for that.